sycl/test-e2e/bindless_images/3_channel_format.cpp

```diff
@@ -0,0 +1,109 @@
+// REQUIRES: cuda
```

Comment:
Why are you limiting this to CUDA adapter?

Reply:
Because it requires extra L0 work, I see you added it here: https://github.com/oneapi-src/unified-runtime/pull/2022/files